.github/workflows/ci.yml

```
@@ -223,7 +223,7 @@ jobs:
 
       - name: Check Salt Releases
         run: |
-          echo '${{ steps.get-salt-releases.outputs.latest-release }}' | jq -C '.'
+          echo \"${{ steps.get-salt-releases.outputs.latest-release }}\" | jq -C '.'
```

```suggestion
          echo '"${{ steps.get-salt-releases.outputs.latest-release }}"' | jq -C '.'
```
And we avoid escapes?